semgrep/tests/e2e/snapshots/test_check/test_basic_rule__absolute/results.json

```diff
@@ -77,5 +77,6 @@
         "line": 3
       }
     }
-  ]
+  ],
+  "structured_errors": []
```

Comment:
Do these have to be keyed separately in the output? Probably easier if users don't need to juggle both errors and structured_errors

Reply:
`errors` will eventually go away, but for backwards compatibility, we need to keep it around for the moment.

Open to a different name

Reply:
Why not just put all errors in errors?

Reply:
because things are currently reading from `errors` expecting a certain format -- I can check how it's currently used. Maybe we can structure our errors to avoid conflicts?

Reply:
Are structured errors just for our rule yaml file? If so we should name it something that makes that more clear. If we intend to have other types of errors in structured error we should merge it with the current error list IMO.

Reply:
no, I'm going to parse the errors from semgrep core and restructure them into a consistent format. This way, all the structure of all errors is clearly defined in the Python code.

Reply:
Yeah so might be worth adding the structured_errors to errors? Not sure we have a documented format for errors anyway?